python/rmm/_lib/device_buffer.pyx

```diff
@@ -34,6 +34,18 @@ cdef class DeviceBuffer:
     def size(self):
         return self.c_obj.get()[0].size()
 
+    @property
+    def __cuda_array_interface__(self):
+        cdef dict intf = {
+            "data": self.data(),
+            "shape": (self.size(),),
```

Comment:
size is a property here
```suggestion
            "shape": (self.size,),
```

Reply:
We have both a `cdef size()` method and a `def size()` property. I think within Cython code the method gets called. Probably a good idea to rename one of them..

Reply:
ah, interesting... Yea should likely rename one

Reply:
Yeah I was trying to use that function. Though no strong feelings about using the function or the property.